staging/src/k8s.io/legacy-cloud-providers/azure/azure_cache.go

```diff
@@ -122,12 +126,16 @@ func (t *timedCache) Get(key string, crt cacheReadType) (interface{}, error) {
 
 	// entry exists
 	if entry.data != nil {
+		// force refresh the cache to get the current accurate data
```

Comment:
shouldn't be `if entry.data != nil && crt != cacheReadTypeForceRefresh`

Reply:
@khenidak updated!